CODEOWNERS

```diff
@@ -5,4 +5,5 @@
 # https://git-scm.com/docs/gitignore#_pattern_format
 
 # Default
-*				@gdbelvin
+*				@google/keytransparency
```

Comment:
That [team](https://github.com/orgs/google/teams/keytransparency/members) only has 4 members, so maybe expand it a bit?

Reply:
Would love to expand. Who else would you suggest that has experience with this repo?